src/App.jsx

```diff
@@ -14,70 +16,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const DEFAULT_VALUE = 'Jam';
+  const [selectedGood, setSelectedGood] = useState(DEFAULT_VALUE);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood
+          ? (
+            <>
+              {`${selectedGood} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => setSelectedGood('')}
+              />
+            </>
+          )
+          : 'No goods selected'}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const selectedProduct = selectedGood === good;
```

Comment:
```suggestion
            const isGoodSelected = selectedGood === good;
```
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/87a3911c-543c-465f-92d2-8ee2b48f8591)